app/helpers/sessions_helper.rb

```diff
@@ -15,4 +23,20 @@ def log_out
     session.delete(:user_id)
     @current_user = nil
   end
+
+  def remember user
+    user.remember
+    cookies.permanent.signed[:user_id] = user.id
+    cookies.permanent[:remember_token] = user.remember_token
+  end
+
+  def authenticated? remember_token
```

Comment:
remove this function
because you defined it on 
https://github.com/awesome-academy/2021-malay-group-1/pull/7/files#diff-9802ca3c9c4cf89904fd44bc114e35ebdf2c5dd3d5b645491e2b253e1afef29bR33

Reply:
shoul i remove line 30?

Reply:
please delete this function on session_helper.rb
because it not use